src/Common/Compat.targets

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
```

Comment:
I would prefer we avoid adding this new targets file and the extra set of copying. For the ApiCompat issue we should figure out what it is keying off of and make sure it is correct. If we have to temporarily set another property then we should do that, but we should understand and fix the ApiCompat issue as well.